bravado/fido_client.py

```diff
@@ -97,6 +97,21 @@ class FidoClient(HttpClient):
     """Fido (Asynchronous) HTTP client implementation.
     """
 
+    def __init__(
+        self,
+        future_adapter_class=None,  # type: typing.Optional[typing.Type[FidoFutureAdapter]]
```

Comment:
Any reason you went for `None` as default value instead of `FidoFutureAdapter`?

Reply:
@sjaensch I can do it in another PR if needed